aws-sns-topic/src/main/java/software/amazon/sns/topic/CreateHandler.java

```diff
@@ -22,7 +23,9 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
         this.logger = logger;
         logger.log("CreateHandler invoked in: "+ request.getStackId()+ " " +request.getLogicalResourceIdentifier());
         final ResourceModel model = request.getDesiredResourceState();
-        final Map<String, String> desiredResourceTags = request.getDesiredResourceTags();
+        final Map<String, String> tagsList = request.getDesiredResourceTags() != null? request.getDesiredResourceTags(): Collections.emptyMap();
+        if (request.getSystemTags() != null)
+            tagsList.putAll(request.getSystemTags());
```

Comment:
What's the difference between normal tags and systemTags?

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html